toolkit/types/cvss/cvss_v4.go

```diff
@@ -62,7 +62,7 @@ func (v *V4) getString(m V4Metric) (string, error) {
 // GetScore implements [Vector].
 func (v *V4) getScore(m V4Metric) byte {
 	b := v.mv[int(m)]
-	if m >= V4ExploitMaturity && b == 0 {
+	if m >= V4ExploitMaturity && (b == 0 /* not present */ || b == 'X' /* not defined */) {
```

Comment:
Are you able to show me where in the spec it says this? The 4.0 spec does not seem to show me a nice formula that v3.1 had

Reply:
Where it says what? That these have specific defaults? That's documented under the relevant tables in the [v4 scoring section](https://www.first.org/cvss/v4.0/specification-document#CVSS-v4-0-Scoring-using-MacroVectors-and-Interpolation).